ctapipe/image/cleaning.py

```diff
@@ -111,6 +113,67 @@ def tailcuts_clean(
         )
 
 
+def time_clustering(
+    geom,
+    image,
+    time,
+    pedestal=1.6,
+    n_noise=2.5,
+    minpts=5,
+    eps=1.0,
+    t_scale=4.0,
+    d_scale=0.25,
+):
+    """
+    Clean an image by selecting pixels which pass a time clustering algorithm using DBSCAN.
```

Comment:
There needs to be an empty line after the initial title line here.

We should add a pre-commit linter for this...